src/screens/Login/index.test.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { MemoryRouter } from 'react-router-dom';
+
+import Login from '../Login';
+
+describe('Check login screen', () => {
+  const queryClient = new QueryClient();
+  const setup = () => {
+    const utils = render(
+      <QueryClientProvider client={queryClient}>
+        <Login />
+      </QueryClientProvider>,
+      { wrapper: MemoryRouter }
+    );
+    const emailInput = utils.getByTestId('email');
```

Comment:
I recommend you to take a look at this https://testing-library.com/docs/queries/about#priority to get the element. Following that priority, you ensure better accessibility 😃